corehq/apps/app_manager/templates/app_manager/app_diff.html

```diff
@@ -0,0 +1,49 @@
+{% extends "style/bootstrap3/base_section.html" %}
```

Comment:
I do think it's worth doing - yes, probably with parameters - because it's going to be annoying to make copy/style changes to either page. Though since I was looking at download_index.html the other day, I don't mind doing it if you have other stuff going on.